pyproject.toml

```
@@ -0,0 +1,139 @@
+# For more information about configuring project metadata for the
+# setuptools build backend, please see
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools"]
+
+[project]
+authors = [
+    { name = "Cybersecurity and Infrastructure Security Agency", email = "github@cisa.dhs.gov" }
+]
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    # Indicate who your project is intended for
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    # Specify the Python versions you support here. In particular, ensure
+    # that you indicate whether you support Python 2, Python 3 or both.
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "celery[redis] >= 5.0.0,<6",
+    "cryptography >= 2.4.2",
+    "defusedxml",
+    "dnspython",
+    "docopt",
+    "mongoengine >= 0.27.0",
+    "pymongo <= 4.10",
+    "python-dateutil >= 2.7.5",
+    "PyYAML >= 6.0.0",
+    "requests >=2.6.1",
+    "schedule >= 0.4.2",
+    "tqdm >= 4.30.0",
+    "xmljson >= 0.2.0",
+]
+description = "Admiral"
```

Improve description:
```suggestion
description = "A distributed certificate transparency log harvester"
```